src/core/components/Paginate/index.js

```diff
@@ -51,10 +51,10 @@ export class PaginateBase extends React.Component {
     // If we can show all of the pages, show them all.
     if (pageCount <= showPages) {
       return makePageNumbers({ start: 1, end: pageCount });
-    // If we are showing less on the right than we should, define the start by the end.
+      // If we are showing less on the right than we should, define the start by the end.
```

Comment:
Not sure the reason for this indentation change; can you restore the comments to be indented as they were before?